AsyncDisplayKit/ASCollectionNode.mm

```diff
@@ -217,6 +218,12 @@ - (_ASCollectionPendingState *)pendingState
   return _pendingState;
 }
 
+- (void)setInverted:(BOOL)inverted
+{
+  self.view.inverted = inverted;
+  self.transform = inverted ? CATransform3DMakeScale(1, -1, 1)  : CATransform3DIdentity;
```

Comment:
We need to implement this flag the same way we implement `delegate` – pass to the pending state or to the view, and that'll make the unused `_inverted` ivar disappear.